xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/transactions/Signer.java

```diff
@@ -51,7 +52,10 @@ static ImmutableSigner.Builder builder() {
    * @return The {@link Address} of the signer account.
    */
   @JsonProperty("Account")
-  Address account();
+  @Default
```

Comment:
This should likely be `@Derived` but we probably want to leave it as `@Default` so that if this value comes back from rippled, we honor the value supplied by the server as opposed to a derived value in Java.

WDYT @nkramer44?

Reply:
Agreed